www/src/components/layout/WithNotifications.tsx

```diff
@@ -42,27 +47,97 @@ export function NotificationsPanel({ closePanel }) {
     { variables: {} },
     data => data.notifications)
 
+  const { data } = useQuery<{me: User}>(ONBOARDING_STATUS)
+  const showOnboardingNotification = useCallback(() => (data?.me?.onboardingChecklist?.dismissed || isOnboardingChecklistHidden()) && data?.me?.onboardingChecklist?.status !== OnboardingChecklistState.Finished, [data])
+
+  if (showOnboardingNotification() && !notifications.length) {
+    return <OnboardingChecklistNotification closePanel={closePanel} />
+  }
+
   if (!notifications.length) {
     return <P padding="medium">You do not have any notifications yet.</P>
   }
 
   return (
-    <InfiniteScroller
-      loading={loadingNotifications}
-      hasMore={hasMoreNotifications}
-      loadMore={fetchMoreNotifications}
-      // Allow for scrolling in a flexbox layout
+    <Flex
       flexGrow={1}
-      height={0}
+      direction="column"
+    >
+      {showOnboardingNotification() && (
+        <OnboardingChecklistNotification closePanel={closePanel} />
+      )}
+      <InfiniteScroller
+        loading={loadingNotifications}
+        hasMore={hasMoreNotifications}
+        loadMore={fetchMoreNotifications}
+      // Allow for scrolling in a flexbox layout
+        flexGrow={1}
+        height={0}
+      >
+        {notifications.map(notification => (
+          <Notification
+            key={notification.id}
+            notification={notification}
+            closePanel={closePanel}
+          />
+        ))}
+      </InfiniteScroller>
+    </Flex>
+  )
+}
+
+function OnboardingChecklistNotification({ closePanel }) {
+  const { setDismissed } = useContext(OnboardingChecklistContext)
+  const [updateChecklist, { loading }] = useMutation(UPDATE_ONBOARDING_CHECKLIST)
+
+  return (
+    <Flex
+      justify="space-between"
+      padding="medium"
+      borderTop="1px solid border-fill-two"
+      borderBottom="1px solid border-fill-two"
     >
-      {notifications.map(notification => (
-        <Notification
-          key={notification.id}
-          notification={notification}
-          closePanel={closePanel}
-        />
-      ))}
-    </InfiniteScroller>
+      <Flex
+        direction="column"
+        gap="xxsmall"
+        height={40}
+      >
+        <P
+          body2
+          bold
+        >Get started
+        </P>
+        <P
+          caption
+          color="text-light"
+        >Get started with our quick start guide.
+        </P>
+      </Flex>
+      <Flex alignItems="center">
+        <Button
+          secondary
+          small
+          loading={loading}
+          onClick={() => {
+            updateChecklist({
+              variables: {
```

Comment:
Also a case where mutation attributes are more readable when added to the hook, I think

Reply:
oh yea, previously there were actually 2 separate mutation calls with different values, that's why it's here.